api/v1alpha1/httproute_types.go

```diff
@@ -93,7 +93,9 @@ const (
 
 // HeaderType constants.
 const (
-	HeaderTypeExact = "Exact"
+	HeaderTypeExact                = "Exact"
```

Comment:
Starting off with just `Exact` (and possibly `Contains`?) seems like a good approach to me.